pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -62,12 +62,10 @@ var _ = Describe("Apply", func() {
 			service.Spec.Type = corev1.ServiceTypeClusterIP
 			service.Spec.ClusterIP = ""
 
-			ops, err := generateServicePatch(cachedService, service)
-			Expect(err).To(BeNil())
-			Expect(ops).ToNot(Equal(""))
+			Expect(generateServicePatch(cachedService, service)).To(BeEmpty())
```

Comment:
Interesting. `generateServicePatch(cachedService, service)` returns two values. What is the behavior in that case? are both the slice and the error expected to "be empty"?

Reply:
In second thought, even if that's the case, I think it's confusing to use it that way.
I'd prefer:
```go
ops, err := generateServicePatch(cachedService, service)
Expect(err).To(BeNil())
Expect(ops).To(BeEmpty())
```

Reply:
https://onsi.github.io/gomega/#handling-errors

> Gomega streamlines this very common use case. Both Ω and Expect accept multiple arguments. The first argument is passed to the matcher, and the match only succeeds if **all subsequent arguments are nil or zero-valued**

Reply:
I still think it's less confusing to not use it like that.
But, if you insist, we can also go with that.
Your call.